packages/expo/android/src/main/java/expo/modules/ReactActivityDelegateWrapper.kt

```diff
@@ -92,15 +96,22 @@ class ReactActivityDelegateWrapper(
       .firstOrNull()
```

Comment:
In future, if modules other than `expo-updates` need to delay app launch, you may need to change this so that all the modules that implement the interface have called their `whenReady()` method before you invoke "loadApp".

Reply:
good catch! i've always been confused to get the app delegate subscribers doing right. in this case maybe we should loop all subscribers and wait for all `whenReady()` to be completed. will address this in a separated pr. thanks!